interface/patient_file/summary/weno_fragment.php

```diff
@@ -0,0 +1,85 @@
+<?php 
+
+/**
+ * Weno Fragment.
+ *
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ * @author    Kofi Appiah <kkappiah@medsov.com>
+ * @copyright Copyright (c) 2023 Kofi Appiah <kkappiah@medsov.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require "../../globals.php";
+
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Twig\TwigContainer;
+
+$twig = new TwigContainer(null, $GLOBALS['kernel']);
```

Comment:
remove above line

Reply:
is above $twig being used anywhere. If not, then remove it and the use TwigContainer line.